packages/cli/commands/init.js

```diff
@@ -82,7 +83,9 @@ const CONFIG_CREATION_FLOWS = {
 };
 
 exports.command = 'init';
```

Comment:
Would it make sense to translate the command itself?

Reply:
I'm not sure the translations will work because we are using abbreviations in some cases. I'm not sure that `init`, short for `initialize`, has an equivalent translation in all languages. 🤔

Reply:
Very good points. I agree 👍